specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/azureFirewall.json

```diff
@@ -596,6 +596,13 @@
             "type": "string"
           }
         },
+        "sourceIpTags": {
```

Comment:
Please add new properties at the end to avoid having breaking changes in SDKs (applies to all new props in this file)

Reply:
Done